### ISSUE TITLE
chandra_repro ACIS/LETG high-order responses

### ISSUE THREAD

**Issue by nplee** [cxcsds/ciao-contrib#336]
Hi Kenny,

This came up at the workshop.  I know there was a bug fix a while ago where for HRC-S/LETG, responses up to the +/- 8th order were generated instead of just the first-order responses.  It looks like for ACIS/LETG, only the first-orders are generated by mktgresp via chandra_repro.

~Nick

**Comment by kglotfelty** [comment]
correct. We create the higher order for HRC since it doesn't have order sorting so you generally always need the higher order responses.  Since ACIS has order sorting you don't need to higher order responses if only fitting 1st order data.
